README.md

```diff
@@ -5,182 +5,212 @@
-* Automatically create and sync vector embeddings for your data ([learn more](#automatically-create-and-sync-llm-embeddings-for-your-data))
-* Search your data using vector and semantic search ([learn more](#search-your-data-using-vector-and-semantic-search))
-* Implement Retrieval Augmented Generation inside a single SQL statement ([learn more](#implement-retrieval-augmented-generation-inside-a-single-sql-statement))
-* Perform high-performance, cost-efficient ANN search on large vector workloads with [pgvectorscale](https://github.com/timescale/pgvectorscale), which complements pgvector.
-
-**Leverage LLMs for data processing tasks:**
-* Retrieve LLM chat completions from models like Claude Sonnet 3.5, OpenAI GPT4o, Cohere Command, and Llama 3 (via Ollama). ([learn more](#usage-of-pgai))
-* Reason over your data and facilitate use cases like classification, summarization, and data enrichment on your existing relational data in PostgreSQL ([see an example](/docs/openai.md)).
+<div align=center>
 
-**Learn more about pgai:** To learn more about the pgai extension and why we built it, read
-[pgai: Giving PostgreSQL Developers AI Engineering Superpowers](http://www.timescale.com/blog/pgai-giving-postgresql-developers-ai-engineering-superpowers).
+[![Auto Create and Sync Vector Embeddings in 1 Line of SQL (pgai Vectorizer)](https://github.com/user-attachments/assets/8a71c774-505a-4335-8b34-cdea9dedb558)](https://youtu.be/ZoC2XYol6Zk?si=atI4XPurEifG0pd5)
 
-**Contributing**: We welcome contributions to pgai! See the [Contributing](/CONTRIBUTING.md) page for more information.
+</div>
 
-# Demo: pgai Vectorizer
 
-[![Auto Create and Sync Vector Embeddings in 1 Line of SQL (pgai Vectorizer)](https://github.com/user-attachments/assets/8a71c774-505a-4335-8b34-cdea9dedb558)](https://youtu.be/ZoC2XYol6Zk?si=atI4XPurEifG0pd5)
+# Quick Start with Ollama
 
-# Supported features in each model
+This section will walk you through the steps to get started with pgai and Ollama and show you the major features of pgai.
 
-| **Model**       | **Tokenize** | **Embed** | **Chat Complete** | **Generate** | **Moderate** | **Classify** | **Rerank** |
-|------------------|:------------:|:---------:|:-----------------:|:------------:|:------------:|:------------:|:----------:|
-| **Ollama**       |              |     ✔️    |         ✔️         |      ✔️      |              |              |           |
-| **OpenAI**       |      ✔️️      |     ✔️    |         ✔️         |              |      ✔️      |              |           |
-| **Anthropic**    |              |           |                    |      ✔️      |              |              |           |
-| **Cohere**       |      ✔️      |     ✔️    |         ✔️         |              |              |      ✔️      |     ✔️     |
+### Setup
 
-# Getting Started
+After checking out the repo, change directory to `examples/docker_compose_pgai_ollama` and run the following command to start the docker compose file:
```

Comment:
I think allowing to copy the compose from the readme is less friction that cloning the whole repository. This e.g. allows people to create a compose in the project that they are currently working on which makes integrating easier later, since they can reuse the same file.

I get that you don't want the ~20 lines of compose here, but we can use a spoler/details tag for that like this:

<details>
<summary> docker-compose.yaml content:</summary>

```yaml
name: pgai
services:
  db:
    image: timescale/timescaledb-ha:pg17
    environment:
      POSTGRES_PASSWORD: postgres
    ports:
      - "5432:5432"
    volumes:
      - data:/home/postgres/pgdata/data
    command: [ "-c", "ai.ollama_host=http://ollama:11434" ]
  vectorizer-worker:
    image: timescale/pgai-vectorizer-worker:v0.3.0
    environment:
      PGAI_VECTORIZER_WORKER_DB_URL: postgres://postgres:postgres@db:5432/postgres
      OLLAMA_HOST: http://ollama:11434
    command: [ "--poll-interval", "5s", "--log-level", "DEBUG" ]
  ollama:
    image: ollama/ollama
volumes:
  data:
```
</details>

Reply:
I opted for a curl command to download instead. Tell me what you think.

Reply:
I guess that's a decent middle ground :D